platforms/corda-ent/charts/node-initial-registration/values.yaml

```diff
@@ -0,0 +1,157 @@
+# Default values for Notary Registration job.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Provide the name of the node
+# Eg. nodeName: carrier-registration
+nodeName:
+
+# This section contains the Enterprise-Corda node metadata.
+metadata:
+  # Provide the namespace for the Corda node.
+  # Eg. namespace: cenm
+  namespace:
+  # Provide the labels to the Corda node.
+  labels:
+
+# Provide image for init container
+image:
+  # Provide the alpine utils image.
+  # NOTE: The alpine image used is the base alpine image with CURL installed.
+  # Eg. initContainerName: hyperledgerlabs/alpine-utils:1.0
+  initContainerName:
+  # Enterprise node image
+  # Eg. adopblockchaincloud0502.azurecr.io/corda_image_ent_4.4:latest
+  nodeContainerName:
+  # Provide the docker-registry secret created and stored in kubernetes cluster as a secret.
+  # Eg. imagePullSecret: regcred
+  imagepullsecret:
+  # Image pull policy
+  # Eg. Always
+  pullPolicy:
+
+# required parameter
+# Accept Corda Enterprise license should be YES.
+acceptLicense: YES
+
+# This section contains the vault related information.
+# NOTE: Make sure that the vault is already unsealed, intialized and configured to
+# use the Kubernetes service account token based authentication.
+vault:
+  # Provide the vault address
+  # Eg. address: http://vault.example.com:8200
+  address: 
+  # Provide the vault role used.
+  # Eg. role: vault-role
+  role:
+  # Provide the authpath configured to be used.
+  # Eg. authpath: entcordacenm
+  authpath:
+  # Provide the service account name autheticated to vault.
+  # NOTE: Make sure that the service account is already created and autheticated to use the vault.
+  # Eg. serviceaccountname: vault-auth
+  serviceaccountname:
+  # Provide the vault path where the  certificates are stored
+  # Eg. certsecretprefix: secret/organisation-name
+  certsecretprefix:
+  # Provide the vault orginsation node folder name where the certificates are stored
+  # Eg. nodePath: carrier
+  nodePath:
+
+# Provide volume related specifications
+volume:
+ # Ex baseDir: /opt/corda
+  baseDir:
+# where is node jar is stored
+jarPath: bin

```

Comment:
these also should be in settings section

Reply:
ok working on this

Reply:
done, moved it to settings section